# Introduction
TsImpute is developed to impute dropouts of scRNA-seq data in a two-step manner and improve down-stream analysis such as cell clustering, differential expression analysis, etc.

# Instruction
Users can just take three steps to use tsImpute:
1. Download tsImpute_0.1.1.tar.gz;
2. Download the example data darmanis.rds and corresponding labels cell_types.rds;
3. Run the script example.R.

# Data availability
Darmanis: [source](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE67835)

Ting: [source](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE51372)

Pollen: [source](https://www.ncbi.nlm.nih.gov/sra/?term=SRP041736)

Huarte: [source](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE148393)

PBMC: [source](https://support.10xgenomics.com/single-cell-gene-expression/datasets/2.1.0/pbmc4k?)

Klein: [source](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE65525)

Baron: [source](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE84133)

Domingo: [source](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE147668)

Chu: [source](https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE75748)
